ext/libxml/libxml.c

```diff
@@ -1071,29 +1063,44 @@ PHP_FUNCTION(libxml_disable_entity_loader)
 /* {{{ Changes the default external entity loader */
 PHP_FUNCTION(libxml_set_external_entity_loader)
 {
+	zval					*callback;
 	zend_fcall_info			fci;
 	zend_fcall_info_cache	fcc;
+	char					*error = NULL;
 
 	ZEND_PARSE_PARAMETERS_START(1, 1)
-		Z_PARAM_FUNC_OR_NULL(fci, fcc)
+		Z_PARAM_ZVAL(callback)
```

Comment:
what was the problem with the original ZPP? We put a lot of effort into using ZPP anywhere where it's possible (in order to make param parsing surely consistent), so I'd be happy if we could keep this one.